ngraph/core/reference/src/runtime/reference/fft.cpp

```diff
@@ -526,6 +542,7 @@ namespace ngraph
                 // not transformed dimensions.
                 for (int64_t outer_idx = 0; outer_idx < outer_size; ++outer_idx)
                 {
+                    std::fill(buffer.begin(), buffer.end(), complex_type{0.0f, 0.0f});
```

Comment:
Why do we need to have this buffer? We don't use modified values.

Reply:
Fixed.